rust/metaplex/program/src/utils.rs

```diff
@@ -604,7 +606,19 @@ pub fn common_redeem_checks(
     }
 
     if AuctionData::get_state(auction_info)? != AuctionState::Ended {
-        return Err(MetaplexError::AuctionHasNotEnded.into());
+        if win_index.is_some() && auction_extended_info.is_some() {
+            if let Some(instant_sale_price) = AuctionDataExtended::get_instant_sale_price(&auction_extended_info.unwrap().data.borrow()) {
+                // we can safely do unwrap here such as existing win_index proves that we can get winner_bid_price
+                let winner_bid_price = AuctionData::get_winner_bid_amount_at(auction_info, win_index.unwrap()).unwrap();
+                if winner_bid_price < instant_sale_price {
```

Comment:
For the OpenEdition auction you can get the bid price from the last_bid field on the BidderMetadata which is present here if overwrite_win_index.is_none(). Might actually be easier than using get_winner_bid_amount_at, which is specific to winners.